policies/cloudwatch_policy.tpl

```diff
@@ -0,0 +1,23 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream"
+      ],
+      "Resource": [
+        "${log_group_arn}"
```

Comment:
Doesn't this need to be `"${log_group_arn}*"` to allow creation of log streams inside that log group?